minard/templates/nearline_summary.html

```diff
@@ -164,9 +164,7 @@
 			        <th class="danger"> Not Launched </th>
 		                <th> {{ program }} </th>
                                 <th><a href="{{ url_for("nearline", run=run) }}">{{ run }}</a></th>
-                                <th>
-                                    <a href="{{ "/monitoring/nearline/logs/%i/%s.log" % (run,program) }}">{{ program }}.log</a>
-                                </th>
+                                <th> - </th>
```

Comment:
Is there a reason to delete the link to the log files from the nearline summary page?

Reply:
This block is for the "not launched" jobs, so the logfile doesn't exist.